### PR TITLE
Kdd fixes

### PR DESCRIPTION
**Pull request by parkererickson**
Fixes from KDD run-through. Async mode on data loaders now in a try-except block. Schema is refreshed during data loader verification. Improvement to TG Cloud check during instantiation (quicker response).

### CODE REVIEW

File: pyTigerGraph/gds/dataloaders.py [hunk 1]
```diff
@@ -415,10 +415,18 @@ def _request_kafka(
...
-            "graph_name": tgraph.graphname,
-            "requestid": resp["request_id"],
-        }
+        try:
+            _stat_payload = {
+                "graph_name": tgraph.graphname,
+                "requestid": resp["request_id"],
+            }
+        except KeyError:
+            print(resp)
```
Comment (tg-bill) [inline pyTigerGraph/gds/dataloaders.py:424]: Is this print left on purpose?

File: pyTigerGraph/gds/dataloaders.py [hunk 2]
```diff
@@ -415,10 +415,18 @@ def _request_kafka(
...
+            if resp["results"][0]["kafkaError"] != '':
+                raise TigerGraphException(
+                    "Error writing to Kafka: {}".format(res[0]["kafkaError"])
+                )
+
```
Comment (tg-bill) [inline pyTigerGraph/gds/dataloaders.py:429]: Need to return here or it will go into the while loop below

File: pyTigerGraph/pyTigerGraphBase.py [hunk 3]
```diff
@@ -128,10 +127,11 @@ def __init__(self, host: str = "http://127.0.0.1", graphname: str = "MyGraph",
 
         self.tgCloud = tgCloud or gcp
         if "tgcloud" in self.netloc.lower():
-            try: # if get request succeeds, using TG Cloud instance provisioned before 6/20/2022
-                self._get(self.host + ":" + str(restppPort) + "/echo/" + graphname, resKey="message")
-            except requests.exceptions.RequestException: # if get request fails, using TG Cloud instance provisioned after 6/20/2022, after new firewall config
+            try: # if get request succeeds, using TG Cloud instance provisioned after 6/20/2022
+                self._get(self.host + "/api/ping", resKey="message")
```
Comment (tg-bill) [inline pyTigerGraph/pyTigerGraphBase.py:131]: Does it fail fast if the instance is not on new tgcloud?

Comment (parkererickson) [reply pyTigerGraph/pyTigerGraphBase.py:131]: Yes it does, confirmed on a box from pre-6/20/2022

File: pyTigerGraph/gds/dataloaders.py (hunk 2 above)
Comment (parkererickson) [reply pyTigerGraph/gds/dataloaders.py:429]: Fixed in next commit.

File: pyTigerGraph/gds/dataloaders.py (hunk 1 above)
Comment (parkererickson) [reply pyTigerGraph/gds/dataloaders.py]: Nope, fixed

**Review by tg-bill — APPROVED** [review]

**Pull request merged**